observation_portal/requestgroups/serializers.py

```diff
@@ -421,6 +450,35 @@ def validate(self, data):
                             ))
                         )
 
+            # Validate the exposure modes
+            if 'exposure' in modes:
```

Comment:
This is a new GenericMode type in configdb, and the two options for muscat both have the exposure_time_* values as required params, so this makes sure they exist. It also checks the value of the exposure_mode is valid, and if not set sets the default value.